projects/applicationinsights-angularplugin-js/src/lib/applicationinsights-angularplugin-error.service.ts

```diff
@@ -1,17 +1,24 @@
-import { ErrorHandler, Injectable } from '@angular/core';
+import { Injectable } from '@angular/core';
 import { IAppInsights } from '@microsoft/applicationinsights-common';
+import { arrForEach } from '@microsoft/applicationinsights-core-js';
+import { IErrorService } from './IErrorService';
 
 @Injectable({
   providedIn: 'root'
 })
-export class ApplicationinsightsAngularpluginErrorService implements ErrorHandler {
+export class ApplicationinsightsAngularpluginErrorService implements IErrorService {
   public static instance: ApplicationinsightsAngularpluginErrorService = null;
   private analyticsPlugin: IAppInsights;
+  private errorServices: IErrorService[];
 
   public set plugin(analyticsPlugin: IAppInsights) {
     this.analyticsPlugin = analyticsPlugin;
   }
 
+  public set errorHandlers(errorServices: IErrorService[]) {
```

Comment:
This might be better as ```addErrorHandler(errorService: IErrorService)``` where it adds to the internally array, as if an application imports multiple components it may not know all of the error handlers.

Reply:
It would also protect the internal implementation from someone passing in something other than an array

Reply:
Updated this to a public function.